_summaries/carion2020end-to-end_1.md

```diff
@@ -1,19 +1,67 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: carion2020end-to-end
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: kraja99
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Overview
```

Comment:
Really great summary! I think you capture the essence of the paper.
I only added one suggestion on Ablations.